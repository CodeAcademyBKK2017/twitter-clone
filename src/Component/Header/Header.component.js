import inputStyles from './Header.css';
import React, {Component} from 'react';
import CircleBG from '../Circle/CircleBG.component';
import CircleNoneBG from '../Circle/CircleNoneBG.component';
import InputBorder from '../InputBorder/InputBorder.component';
import Icon from 'react-ionicons';
import className from 'classname';

class Header extends Component {
  state = {
    menuVisibility: false
  }
  toggleMenu =() => {
    this.setState({menuVisibility: !this.state.menuVisibility});
  }
  render () {
    return (
      <div className={className(inputStyles.header, inputStyles.shadow)}>
        { 
          this.state.menuVisibility ?
            <div className={className(inputStyles.sidemenu)}>
              <div className={inputStyles.borderStyle}>
                <input className={inputStyles.inputStyle} placeholder="Search"/>
                <Icon className={inputStyles.colorStyle} icon='ios-search'/>
              </div>
              <div className={inputStyles.name}>Nanny</div>
            </div> : null
        }
        
        <div className={className(inputStyles.show)} onClick={this.toggleMenu}>
          <Icon icon="ios-menu"/></div> 
        <div className={className(inputStyles.flex4, inputStyles.contentLeft)}>
        
          <CircleBG iconName={'logo-twitter'} color='white'/>

          <div className={inputStyles.divide}/>

          <CircleNoneBG color='#5f5f5f' iconName={'ios-notifications'}/>
          <CircleNoneBG color='#5f5f5f' iconName={'ios-mail'}/>
          <CircleNoneBG color='#5f5f5f' iconName={'ios-grid-outline'}/>

        </div>

        <div className={inputStyles.flex1}><Icon fontSize='35px' color='#1e90ff' icon='logo-twitter'/></div>

        <div className={className(inputStyles.flex4, inputStyles.contentRight)}>
          <InputBorder />
          <CircleNoneBG color='red' iconName={'md-heart'}/>
        </div>
        
      </div>

    );
  }
}

export default Header;
