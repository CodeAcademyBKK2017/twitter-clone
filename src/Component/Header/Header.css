  .Box-Header {  
    display: flex;  
    height: 60px;
    position: fixed;
    top:0;
    width:100%;    
    justify-content: center;
    align-items: center;
  } 
  
  .Header4 {
    flex-grow: 4;
  } 
  
  .Header1 {
    flex-grow: 1;
    display: flex;
    justify-content: center;
    align-items: center;
  }

  .logo {
    color:#80A8CC;
    size : 2x;
  }
  
