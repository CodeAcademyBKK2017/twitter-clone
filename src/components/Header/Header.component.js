import React, {Component} from 'react';
import classNames from 'classnames';
import headerStyle from './Header.style.css';
import Ionicon from 'react-ionicons';
import Circle from '../Circle/Circle.component';
import Search from '../Search/Search.component';

class Header extends Component {
  render () {
    return (
      <div>
        <div className={classNames(headerStyle.headerMobile)}>
          <div className={classNames(headerStyle.headerMobile__menuIconContainer)}>
            <Ionicon icon="ios-menu" fontSize="22px" className={classNames(headerStyle.headerMobile__menuIcon)} />
          </div>
          <div className={classNames(headerStyle.headerMobile__logoContainer)}>
            <center><Ionicon icon="logo-twitter" fontSize="35px" color="#81a9cb" /></center>
          </div>
        </div>
        <div className={classNames(headerStyle.sideMenu__panel)}>
          <Search />
        </div>
        <div className={classNames(headerStyle.header)}>
          <div className={classNames(headerStyle.left)}>
            <div className={classNames(headerStyle.circleContainer, headerStyle.circleContainer__first, headerStyle.circleContainer__borderRight)}>
              <Circle iconName="logo-twitter" type="brand" />
            </div>
            <div className={classNames(headerStyle.circleContainer)}>
              <Circle iconName="ios-notifications-outline" />
            </div>
            <div className={classNames(headerStyle.circleContainer)}>
              <Circle iconName="ios-mail-outline" />
            </div>
            <div className={classNames(headerStyle.circleContainer, headerStyle.active)}>
              <Circle iconName="ios-globe" />
            </div>
          </div>
          <div className={classNames(headerStyle.logo)}><Ionicon icon="logo-twitter" fontSize="35px" color="#81a9cb" /></div>
          <div className={classNames(headerStyle.right)}>
            <Search />
            <div className={classNames(headerStyle.circleContainer, headerStyle.circleContainer__user)}>
              <Circle iconName="ios-ionitron-outline" />
            </div>
          </div>
        </div>
      </div>
    );
  }
}

export default Header;