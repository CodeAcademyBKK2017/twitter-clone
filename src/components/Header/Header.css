.background_white{
    background-color: white;
}

.box_shadow {
    box-shadow: 0 2px 2px 0 rgba(0,0,0,0.08), 0 0 0 1px rgba(0,0,0,0.04);
}

.header {
    position: fixed;
    left: 0;
    right: 0;
    height: 60px;
    top: 0;
    display: flex;
}

.left_head{
    flex: 4;
    display: flex;
    align-items: center;
    justify-content: flex-start;
}
.center_head{
    flex: 1;
    color: #80A8CC;
    display: flex;
    align-items: center;
    justify-content: center;
}
.right_head{
    flex: 4;
    display: flex;
    align-items: center;
    justify-content: flex-end;
}
.right_head_text {
    padding: 5px;
    border: 1px solid gray;
    display: inline-flex;
    border-radius: 10px;
}
.right_head_text > input {
    border: none;
    outline: none;
    background: none;
    width: 20vw;
}
.right_head_icon {
    padding-right: 10px;
}

.head_first_icon {
    padding: 0 10px 0 0;
    border-right: 1px solid black;
    height: 100%;
    align-items: center;
    display: flex;
}
.head_second_icon {
    height: 100%;
    align-items: center;
    display: flex;
}

.header_mobile {
    display: none;
}
.menuBar {
    left: -80vw;
}

@media only screen and (max-width: 768px) {
    .header {
        display: none;
    }
    .header_mobile {
        position: fixed;
        left: 0;
        right: 0;
        height: 60px;
        top: 0;
        display: flex;
    }

    .menuBar{
        /* display: inline-flex; */
        bottom: 0;
        top: 60px;
        position: fixed;
        /* width: 80vw; */
        background-color: #F2F2F2;
        padding: 5px 5px 5px 5px;
        transition: all 0.5s;
    }
    .searchTextBox{
        /* display: inline-flex; */
        width: 100%;
    }
    .right_head_text > input{
        /* display: inline-flex; */
        width: 100%;
    }
    .bottomText{
        position: absolute;
        bottom: 0;
        left: 0;
        right: 0;
        text-align: center;
        padding: 10px;
        border-top: 2px solid #ccc;
    }

    .activeMenu{
        left: 0vw;
        width: 80vw;
    }
}