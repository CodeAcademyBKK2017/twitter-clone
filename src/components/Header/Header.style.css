.header {
    position: fixed;
    left: 0;
    right: 0;
    top: 0;
    bottom: 0;
    height: 60px;
    background-color: white;
    color: black;
    box-shadow: 0 2px 2px 0 rgba(0,0,0,0.08), 0 0 0 1px rgba(0,0,0,0.04);
    display: flex;
}
.left {
    flex: 4;
    display: flex;
    align-items: center;
}
.logo {
    flex: 1;
    height: 100%;
    color: #80A8CC;
    display: flex;
    align-items: center;
    justify-content: center;
}
.right {
    flex: 4;
    display: flex;
    align-items: center;
    justify-content: flex-end;
}
.circleContainer {
    height: 100%;
    display: inline-flex;
    align-items: center;
    justify-content: center;
}
.circleContainer__first {
    width: 55px;
    margin-right: 5px;
}
.circleContainer__user {
    width: 55px;
}
.circleContainer__borderRight {
    border-right: 1px solid #f8f4ee;
}
.circleContainer__borderBottom {
    border-bottom: 2px solid #81a9cb;
}
.circleContainer+.circleContainer {
    width: 45px;
}