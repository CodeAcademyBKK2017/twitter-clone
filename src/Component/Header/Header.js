import './Header.css';
import Circle from '../Circle/Circle'
import React, {Component} from 'react';
import {Icon} from 'react-fa';

class Header extends Component {
  render () {
    return (     
        <div className='Box-Header shadow box-bg'>
          <div className='Header4'><Circle name='twitter'/></div>
          <div className='Header1'><Icon size='2x' className='logo' name="twitter"/></div>
          <div className='Header4'>c</div>        
        </div>      
    );
  }
}

export default Header;
