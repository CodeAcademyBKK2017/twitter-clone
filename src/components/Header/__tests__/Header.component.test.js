import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../Header.component';

it('Cell: Snapshot test', () => {
  const snapshot = renderer.create(<Header />).toJSON();
  expect(snapshot).toMatchSnapshot();
});