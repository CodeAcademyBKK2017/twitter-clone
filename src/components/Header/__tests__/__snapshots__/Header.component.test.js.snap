// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cell: Snapshot test 1`] = `
<div>
  <div
    className=""
  >
    <div
      className=""
    >
      <svg
        className="sc-bdVaJa fUuvxv"
        fill="#000000"
        height="22px"
        onClick={undefined}
        rotate={0}
        style={Object {}}
        viewBox="0 0 1024 1024"
        width="22px"
      >
        <path
          d="M128 288h768v64h-768v-64z M128 480h768v64h-768v-64z M128 672h768v64h-768v-64z"
        />
      </svg>
    </div>
    <div
      className=""
    >
      <center>
        <svg
          className="sc-bdVaJa fUuvxv"
          fill="#81a9cb"
          height="35px"
          onClick={undefined}
          rotate={0}
          style={Object {}}
          viewBox="0 0 1024 1024"
          width="35px"
        >
          <path
            d="M984 219c-34.8 15.4-72 25.8-111.2 30.6 40-24 70.8-62 85.2-107.2-37.4 22.2-78.8 38.4-123 47-35.4-37.8-85.8-61.4-141.4-61.4-107 0-193.6 86.8-193.6 193.8 0 15.2 1.6 30 5 44.2-161-8-303.8-85.2-399.2-202.6-16.6 28.6-26.2 62-26.2 97.4 0 67.2 34.4 126.6 86.4 161.4-32-0.8-62-9.6-88-24.2 0 0.8 0 1.6 0 2.4 0 94 66.8 172.2 155.4 190-16.2 4.4-33.4 6.8-51 6.8-12.4 0-24.6-1.2-36.4-3.6 24.6 77 96.2 133 181 134.6-66.2 52-149.8 83-240.6 83-15.6 0-31-1-46.2-2.8 85.4 55.6 187.2 87.6 296.4 87.6 356.6 0 551.4-295.4 551.4-551.6 0-8.4-0.2-16.8-0.6-25 37.8-27.4 70.6-61.4 96.6-100.4z"
          />
        </svg>
      </center>
    </div>
  </div>
  <div
    className=""
  >
    <div
      className=""
    >
      <input
        className=""
        placeholder="Search"
        type="text"
      />
      <svg
        className="sc-bdVaJa fUuvxv"
        fill="#000000"
        height="22px"
        onClick={undefined}
        rotate={0}
        style={Object {}}
        viewBox="0 0 1024 1024"
        width="22px"
      >
        <path
          d="M896.6 849.4l-226.6-226.8c41.6-52 66.6-118.2 66.6-190.2 0-168.2-136.2-304.4-304-304.4-168 0-304 136.4-304 304.4s136.2 304.4 304 304.4c72.4 0 138.8-25.4 191-67.6l226.4 226.8 46.6-46.6zM240.2 625.2c-51.4-51.4-79.6-119.8-79.6-192.6s28.4-141.2 79.6-192.6 119.8-80 192.4-80c72.6 0 141 28.4 192.4 79.8s79.6 119.8 79.6 192.6-28.4 141.2-79.6 192.6c-51.4 51.4-119.8 79.8-192.4 79.8-72.6 0.2-141-28.2-192.4-79.6z"
        />
      </svg>
    </div>
  </div>
  <div
    className=""
  >
    <div
      className=""
    >
      <div
        className=""
      >
        <div
          className=""
        >
          <svg
            className="sc-bdVaJa fUuvxv"
            fill="#000000"
            height="22px"
            onClick={undefined}
            rotate={0}
            style={Object {}}
            viewBox="0 0 1024 1024"
            width="22px"
          >
            <path
              d="M984 219c-34.8 15.4-72 25.8-111.2 30.6 40-24 70.8-62 85.2-107.2-37.4 22.2-78.8 38.4-123 47-35.4-37.8-85.8-61.4-141.4-61.4-107 0-193.6 86.8-193.6 193.8 0 15.2 1.6 30 5 44.2-161-8-303.8-85.2-399.2-202.6-16.6 28.6-26.2 62-26.2 97.4 0 67.2 34.4 126.6 86.4 161.4-32-0.8-62-9.6-88-24.2 0 0.8 0 1.6 0 2.4 0 94 66.8 172.2 155.4 190-16.2 4.4-33.4 6.8-51 6.8-12.4 0-24.6-1.2-36.4-3.6 24.6 77 96.2 133 181 134.6-66.2 52-149.8 83-240.6 83-15.6 0-31-1-46.2-2.8 85.4 55.6 187.2 87.6 296.4 87.6 356.6 0 551.4-295.4 551.4-551.6 0-8.4-0.2-16.8-0.6-25 37.8-27.4 70.6-61.4 96.6-100.4z"
            />
          </svg>
        </div>
      </div>
      <div
        className=""
      >
        <div
          className=""
        >
          <svg
            className="sc-bdVaJa fUuvxv"
            fill="#000000"
            height="22px"
            onClick={undefined}
            rotate={0}
            style={Object {}}
            viewBox="0 0 1024 1024"
            width="22px"
          >
            <path
              d="M576 800c0 34-28.8 64-64 64s-64.4-30-64.4-64h-32c0 52 43.2 95.8 96.2 95.8s96.2-43.8 96.2-95.8h-32z M747.8 458.4c0-198-98-265.6-189.8-281 0-1.2 0.2-2.2 0.2-3.4 0-25.4-20.6-46-46-46s-46 20.6-46 46c0 1.2 0 2.4 0.2 3.6-91.8 15.6-189.8 83.8-189.8 281.4-0.4 231-45.4 245-116.6 309h704c-70.8-64-116.2-78.2-116.2-309.6zM242.2 736c14.4-16 27.4-34.8 37.8-60 19.6-47.4 28.2-114.6 28.2-217.4 0-82.6 18-146.2 53.6-189 27.2-32.8 67.4-55.4 110.8-61 33.6-4.2 44.4-4.8 79.4 0 41.6 5.8 83.2 28 110.4 60.8 35.4 42.8 53.4 106.4 53.4 189 0 102.8 8.8 169.8 28.4 217.4 10.4 25.2 23.4 44.2 38 60.2h-540z"
            />
          </svg>
        </div>
      </div>
      <div
        className=""
      >
        <div
          className=""
        >
          <svg
            className="sc-bdVaJa fUuvxv"
            fill="#000000"
            height="22px"
            onClick={undefined}
            rotate={0}
            style={Object {}}
            viewBox="0 0 1024 1024"
            width="22px"
          >
            <path
              d="M128 256v512h768v-512h-768zM512 535.8l-325.6-247.8h651.2l-325.6 247.8zM160 736v-427.8l230.2 175.2-136.2 154.6 4 4 157.8-139.2 96.2 73.2 96.2-73.2 157.8 139.2 4-4-136.2-154.8 230.2-175v427.8h-704z"
            />
          </svg>
        </div>
      </div>
      <div
        className=""
      >
        <div
          className=""
        >
          <svg
            className="sc-bdVaJa fUuvxv"
            fill="#000000"
            height="22px"
            onClick={undefined}
            rotate={0}
            style={Object {}}
            viewBox="0 0 1024 1024"
            width="22px"
          >
            <path
              d="M512 96c-0.2 0-0.2 0-0.4 0 0 0 0 0-0.2 0s-0.4 0-0.4 0c-229.4 0.6-415 186.6-415 416s185.6 415.4 415 416c0.2 0 0.4 0 0.4 0s0.2 0 0.2 0 0.2 0 0.4 0c229.8 0 416-186.2 416-416s-186.2-416-416-416zM512 894.8c-0.2 0-0.2 0-0.4 0 0 0 0 0-0.2 0s-0.4 0-0.4 0c-210.8-0.6-381.8-171.8-381.8-382.8s170.8-382.2 381.8-382.6c0.2 0 0.2 0 0.4 0 0 0 0.2 0 0.2 0 0.2 0 0.2 0 0.4 0 211.4 0 382.8 171.4 382.8 382.8 0 211.2-171.4 382.6-382.8 382.6z M644.6 343.6c-37.4 9-76 14.4-115.8 15.6v136.2h135.2c-1.2-56.2-8.2-107.2-19.4-151.8z M528.6 165v163.8c37-1.2 73-6 107.6-14.4-26.2-82.8-67-137.2-107.6-149.4z M386.8 314.2c35 8.4 71.2 13.4 108.4 14.6v-164.2c-40.6 11.8-82 66.2-108.4 149.6z M761.2 266.8c-45.2-46-103-79.6-167.8-95 29.6 30.6 54.4 77.4 72.2 134.6 33.4-10.2 65.4-23.4 95.6-39.6z M694.6 495.4h166.6c-3.6-77.6-32.6-148.8-79-205.2-34.2 18.8-70.4 34.2-108.2 45.6 12.2 48 19.4 102.2 20.6 159.6z M359 495.4h136.4v-136.2c-40.2-1.2-79.2-6.6-116.8-15.8-11.4 44.6-18.4 95.8-19.6 152z M664 528.6h-135.4v136c39.8 1.2 78.6 6.4 116 15.6 11.2-44.4 18.2-95.6 19.4-151.6z M378.6 680.4c37.6-9.2 76.6-14.6 116.8-15.8v-136h-136.4c1.2 56.2 8.2 107.4 19.6 151.8z M495.4 859.2v-164.2c-37.2 1.2-73.6 6.2-108.6 14.6 26.4 83.4 67.8 138 108.6 149.6z M593.4 852.2c64.8-15.6 122.6-49 168-95.2-30.2-16.2-62.2-29.4-95.6-39.6-17.8 57.4-42.8 104-72.4 134.8z M429.4 172.2c-64.2 15.6-121.6 49-166.6 94.8 30 16 61.6 29.2 94.6 39.2 17.8-57 42.6-103.4 72-134z M674 688.2c38 11.4 74.2 26.8 108.4 45.6 46.4-56.4 75.4-127.6 79-205.2h-166.6c-1.4 57.4-8.6 111.4-20.8 159.6z M528.6 695v164c40.6-12.2 81.4-66.6 107.6-149.6-34.6-8.4-70.6-13.2-107.6-14.4z M349.2 335.6c-37.6-11.4-73.6-26.6-107.4-45.4-46.4 56.4-75.4 127.6-79 205.2h165.8c1-57.6 8.4-111.8 20.6-159.8z M328.4 528.6h-165.8c3.6 77.6 32.6 148.8 79 205.2 33.8-18.6 69.8-34 107.4-45.4-12-48-19.4-102.2-20.6-159.8z M262.6 757c45 45.8 102.4 79.2 166.8 94.8-29.4-30.6-54.2-77.2-72-134.2-33 10.2-64.8 23.4-94.8 39.4z"
            />
          </svg>
        </div>
      </div>
    </div>
    <div
      className=""
    >
      <svg
        className="sc-bdVaJa fUuvxv"
        fill="#81a9cb"
        height="35px"
        onClick={undefined}
        rotate={0}
        style={Object {}}
        viewBox="0 0 1024 1024"
        width="35px"
      >
        <path
          d="M984 219c-34.8 15.4-72 25.8-111.2 30.6 40-24 70.8-62 85.2-107.2-37.4 22.2-78.8 38.4-123 47-35.4-37.8-85.8-61.4-141.4-61.4-107 0-193.6 86.8-193.6 193.8 0 15.2 1.6 30 5 44.2-161-8-303.8-85.2-399.2-202.6-16.6 28.6-26.2 62-26.2 97.4 0 67.2 34.4 126.6 86.4 161.4-32-0.8-62-9.6-88-24.2 0 0.8 0 1.6 0 2.4 0 94 66.8 172.2 155.4 190-16.2 4.4-33.4 6.8-51 6.8-12.4 0-24.6-1.2-36.4-3.6 24.6 77 96.2 133 181 134.6-66.2 52-149.8 83-240.6 83-15.6 0-31-1-46.2-2.8 85.4 55.6 187.2 87.6 296.4 87.6 356.6 0 551.4-295.4 551.4-551.6 0-8.4-0.2-16.8-0.6-25 37.8-27.4 70.6-61.4 96.6-100.4z"
        />
      </svg>
    </div>
    <div
      className=""
    >
      <div
        className=""
      >
        <input
          className=""
          placeholder="Search"
          type="text"
        />
        <svg
          className="sc-bdVaJa fUuvxv"
          fill="#000000"
          height="22px"
          onClick={undefined}
          rotate={0}
          style={Object {}}
          viewBox="0 0 1024 1024"
          width="22px"
        >
          <path
            d="M896.6 849.4l-226.6-226.8c41.6-52 66.6-118.2 66.6-190.2 0-168.2-136.2-304.4-304-304.4-168 0-304 136.4-304 304.4s136.2 304.4 304 304.4c72.4 0 138.8-25.4 191-67.6l226.4 226.8 46.6-46.6zM240.2 625.2c-51.4-51.4-79.6-119.8-79.6-192.6s28.4-141.2 79.6-192.6 119.8-80 192.4-80c72.6 0 141 28.4 192.4 79.8s79.6 119.8 79.6 192.6-28.4 141.2-79.6 192.6c-51.4 51.4-119.8 79.8-192.4 79.8-72.6 0.2-141-28.2-192.4-79.6z"
          />
        </svg>
      </div>
      <div
        className=""
      >
        <div
          className=""
        >
          <svg
            className="sc-bdVaJa fUuvxv"
            fill="#000000"
            height="22px"
            onClick={undefined}
            rotate={0}
            style={Object {}}
            viewBox="0 0 1024 1024"
            width="22px"
          >
            <path
              d="M935.4 538.4c-8.6-61.8-28.8-114.6-67.6-114.6l-15.2 0.2c-61.2-124-188.4-207-333.2-207-7 0-14 0.2-21 0.6-171.6 9.6-309.4 133.4-343.4 293.6l-22.8 3.2c-34.4 4.8-53.8 69.6-43.2 145 10 72.4 44.2 128 77.2 128 1.4 0 2.8 0 4-0.2l31.4-4.4c65.8 107 184.2 177.4 317 177.4 7 0 14-0.2 21-0.6 170.4-9.4 307.6-131.8 342.8-290.4l14.8-0.8c30.4-5 47.6-62.8 38.2-130zM212.4 748.8l-46.2 6.4c-2.2-0.6-11.2-6-21.6-24.4-11.4-20-19.8-47-23.8-76s-3.2-57.2 2.2-79.6c5-20.4 12.2-28 14.2-29.4l43.2-6c14.4 25.8 27.2 60 33 98.6 6 41.8 8 81.4-1 110.4zM902.2 610.6c-3.6 16-8.6 23.6-11 26l-35 2-5.2 23.6c-8 35.8-21.6 69.8-40.6 101.4-18.8 31-42.2 58.8-69.8 82.4-57.2 49.2-127.4 77.4-202.8 81.6-6.4 0.4-12.8 0.6-19.2 0.6-58.8 0-116.8-15.4-167.8-44.4-49.8-28.4-92-69-121.8-117.8l-10.2-16.6c16-23.8 25.6-63.2 19.2-115-5.6-45.2-30.8-81-54.8-101.6l3.2-14.8c15.2-71.6 54.2-137.4 109.8-185.8 57.4-49.8 128.2-78.2 204.2-82.4 6.4-0.4 13.6-0.6 19.6-0.6 64.4 0 126.4 18 180.6 52 53.2 33.4 95.8 80.8 123.4 137l7.2 15c16.2 38 25 74.2 27.8 115 3-50.6-4.2-85.4-14.2-112l23-0.2c6.8 0 13.8 11.8 16.6 17 8.4 16 15 40.2 19.2 70.2 3.4 24.2 3 48.2-1.4 67.4z M543.961 698.821c16.985-0.944 31.519 12.059 32.464 29.044s-12.059 31.519-29.044 32.464c-16.985 0.944-31.519-12.059-32.464-29.044s12.059-31.519 29.044-32.464z M705.6 689.6c-17 1-30 15.4-29 32.4s15.4 30 32.4 29c17-1 30-15.4 29-32.4-0.8-17-15.4-30-32.4-29z M336 147.2c4.2 2.8 10 4.8 17.2 6 5 0.8 10.6 1.2 16.6 1.2 2.4 0 4.8 0 7.2-0.2 2 0 3.8-0.2 5.8-0.4-0.8-0.8-1.6-1.8-2.4-2.6-0.8 0-1.8 0-2.6 0s-1.6 0-2.4 0c-7.6 0-14.8-0.6-20.8-1.8-7-1.4-13.4-4.2-16.8-6.8-3.2-2.8-2.6-7.4-2.6-7.4 0.6-5.2 4.8-9.6 10.8-14.8 5.4-4.6 12.8-9.4 21.6-14l6.2-3c-1.6 6.6-1.8 13.8-0.4 21 4.2 21.8 21.8 37.8 42.8 40.8l9.4 46.2 31.6-6.4-9.2-45.6c18.8-10.2 29.6-31.6 25.6-53.8-1.4-7-4-13.4-7.8-19-3.6-0.6-8.6-1-14.4-1-11.2 0-26.2 1.2-44 5.2 2.2-0.6 4.2-1.4 6.4-2 15.2-4.2 29-7 43-8.4 17.2-2 30-0.6 33.2-0.2s6 1 8.6 1.8c-1.6 2.4-2.4 5.4-2.4 8.4 0 6.6 4.2 12.4 10 14.8-0.4 0.6-1 1.2-1.4 1.8-1.6 1.8-3.6 3.8-5.8 5.6s-4.4 3.6-7 5.4l-13 7.6c-0.2 1-0.4 2-0.6 3 0.2 0 0.4-0.2 0.4-0.2l16.6-9.4c2.4-1.6 4.8-3.2 6.8-4.8 2.4-1.8 4.4-3.6 6.2-5.6 0.6-0.6 1.4-1.4 2-2 0.6 0 1.2 0 1.8 0 8.8 0 16-7.2 16-16s-7.2-16-16-16c-3.2 0-6.2 1-8.8 2.6-3.6-1.6-7.2-2.6-12.8-3.6s-23.6-2.4-39.2-0.6c-6.2-4.6-17.8-8.4-27.8-8.4-3.2 0-6.2 0.2-9.4 0.8-15.8 3-28.4 13-35.4 26 0 0-23.4 9-39.2 23.4-6.4 5.8-10 12.8-10.6 16.6-1.4 5.8 0.6 11.6 7 15.8zM423.2 92.6c11.6 0 21 9.4 21 21s-9.4 21-21 21c-11.6 0-21-9.4-21-21s9.4-21 21-21z"
            />
          </svg>
        </div>
      </div>
    </div>
  </div>
</div>
`;
